client/start_spring_test_server.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cd ../server
+./mvnw spring-boot:run -Dspring-boot.run.profiles=testing
```

Comment:
This file is used to start the backend server during CI tests. It's called when running `npm run start-test-backend`. 

`start-test-backend` script in `package.json`:
```
"start-backend": "./start_spring_server.sh"
```

Initially the script looked like:
```
"start-backend": "../server/mvnw spring-boot:run -Dspring-boot.run.profiles=testing"
```

However, the problem was that `./mvnw` was looking for something in the `client` directory. `client` is the directory `package.json` lives. So, it seems that `./mvnw` has to be called from the `server` directory. npm scripts are not bash scripts, so it cannot use `cd`.

As a result I ended up with running a bash script.